src/components/StyleGuide/FilteringApplyingClearing.mdx

```diff
@@ -0,0 +1,106 @@
+import { Meta } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Style Guide/Filtering/Applying & Clearing Filters" />
+
+# Applying and clearing filters
+
+## Table of contents
+
+- {<Link href="#methods" target="_self">Methods</Link>}
+- {<Link href="#applying-filters" target="_self">Applying filters</Link>}
+- {<Link href="#clearing-filters" target="_self">Clearing filters</Link>}
+- {<Link href="#populating-results" target="_self">Populating results</Link>}
+
+## Methods
+
+Filters should be applied and cleared using one of two methods: live filtering or batch filtering.
+
+### Live filtering
+
+- Data is updated in real-time as the user selects filtering options.
+- This is the recommended method for applying filters because:
+  - Users tend to expect and are familiar with real-time filtering due to its
+    prevalence in e-commerce and elsewhere.
+  - We have observed in previous research that users tend to miss `Apply`
+    buttons, particularly when they sit out of sight on the page.
+  - Batch filtering comes at the risk of returning zero results after which
+    users have to engage in a trial-and-error process to get useful results.
+
+### Batch filtering
+
+- The user selects multiple filtering options and the data is updated after the
+  user takes the additional action of clicking the "Apply filters" button
+  included in the `filter bar`.
+- This method is recommended for:
+  - very heavy data sets
+  - low performing apps
+
+## Applying filters
+
+When filters are applied, the following should be addressed:
+
+- update the `active filters` list based on the filter options that have been
+  selected
+- update the `results` list based on the filter options that have been selected
+- update the `total results` heading to reflect the number of items in the
+  updated `results` list
+
+## Clearing filters
+
+Filters can be cleared using various methods within the page structure,
+including global methods as well as methods for clearing individual filters.
```

Comment:
I'm curious: If a user searches for a term and applies some filters, but then continues typing in the search box without clearing the filters, should the filters be reset? Not sure if this is a accessibility question or a product question.